pkg/task/v1alpha1/runtask_command_test.go

```diff
@@ -115,7 +119,7 @@ func TestSelectsQueryCommandResult(t *testing.T) {
 		"102": {[]string{}, nil},
 		"103": {[]string{"{.Name}"}, map[string]interface{}{"s0": "my-cstor-rep"}},
 		"104": {[]string{"{.Spec.TargetIP}", "{.Spec.Capacity}"}, map[string]interface{}{"s0": "20.10.10.10", "s1": "40Gi"}},
-		"105": {[]string{"{..TargetIP}", "{..Capacity}"}, map[string]interface{}{"s0": "20.10.10.10", "s1": "40Gi"}},
+		"105": {[]string{"{..TargetIP}", "{..Capacity}"}, map[string]interface{}{"s0": "20.10.10.10", "s1": []string{"40Gi", "{6K 6K}"}}},
```

Comment:
@sonasingh46 Can you please try with `"{.Status.Capacity}"` instead of `"{..Capacity}"`

Even though there is no error, above path will be deterministic.